code/datums/quirks/negative.dm

```diff
@@ -10,17 +10,17 @@
 	lose_text = "<span class='notice'>Your back feels better.</span>"
 	medical_record_text = "Patient scans indicate severe and chronic back pain."
 	hardcore_value = 4
-	mail_goodies = list(/obj/item/cane)
+	mail_goodies = list(/obj/item/storage/briefcase)
 	var/datum/weakref/backpack
 
 /datum/quirk/badback/add()
 	var/mob/living/carbon/human/human_holder = quirk_holder
 	var/obj/item/storage/backpack/equipped_backpack = human_holder.back
 	if(istype(equipped_backpack))
 		quirk_holder.add_mood_event("back_pain", /datum/mood_event/back_pain)
-		RegisterSignal(human_holder.back, COMSIG_ITEM_POST_UNEQUIP, .proc/on_unequipped_backpack)
+		RegisterSignal(human_holder.back, COMSIG_ITEM_POST_UNEQUIP, PROC_REF(on_unequipped_backpack))
```

Comment:
I think you messed up a merge commit here? These are already PROC_REFs on master.

Reply:
This was taken care off after MCs were resolved.